app/src/main/java/dev/sebastiano/bundel/MainScreenWithBottomNav.kt

```diff
@@ -60,7 +60,7 @@ import java.time.LocalDateTime
 internal fun MainScreenWithBottomNav(
     lifecycle: Lifecycle,
     repository: DataRepository,
-    preferences: Preferences,
+    preferences: dev.sebastiano.bundel.preferences.Preferences,
```

Comment:
Import?

Reply:
done. Missed this with the auto-refactor 👍